.rubocop.yml

```diff
@@ -67,6 +67,12 @@ Style/GuardClause:
 RSpec/ContextWording:
   Enabled: false
 
+RSpec/ExampleLength:
+  Max: 10
```

Comment:
Dez não é um pouco demais pra exemplos, qual é o default do rubocop?

Reply:
Default é 5

Reply:
Na minha opinião, seria bom deixar 5 mesmo, para incentivar escrevermos exemplos mais claros

Reply:
Acho 5 o suficiente também :smile:

Reply:
Não concordo que a clareza se reflita no número de linhas do `it`. A gente acaba apenas movendo as linhas pra outro lugar, talvez até dificultando mais o entendimento

Reply:
Sim, concordo. Dá pra colocar algo como 7, que acho que é um número ok